instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/internal/AlternativeNamePortGetter.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter.net.internal;
+
+import javax.annotation.Nullable;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public interface AlternativeNamePortGetter<REQUEST> {
```

Comment:
maybe `FallbackNamePortGetter`, or something to make it clearer it's "in addition to"? I was initially reading it as an alternative to the default getter methods and got myself confused